python/setup.py

```diff
@@ -2,28 +2,30 @@
 from version import get_git_version
 
 setup_args = dict(
-  name = 'swiftnav',
-  version = get_git_version(),
-  description = 'Python bindings to the libswiftnav library.',
-  license = 'LGPLv3',
-  url = 'http://www.swiftnav.com',
-  author = 'Swift Navigation Inc.',
-  author_email = 'dev@swiftnav.com',
-  maintainer = 'Swift Navigation',
-  maintainer_email = 'dev@swiftnav.com',
-  packages = ['swiftnav'],
+    name='swiftnav',
```

Comment:
I think if `hanging indents should add a level` and we define a level as indented two spaces, then hanging indents are an additional 2 spaces. @valeri-atamaniouk feel free to update the linked hackpad above. @fnoble do you have an opinion?